packages/code-snippet-block/src/helpers.ts

```diff
@@ -0,0 +1,14 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+export const isPxSuffixMissing = (value: string) => !value.includes('px'); // TODO: remove isPxSuffixMissing - use appendUnit after fix
+
+export const appendPxSuffix = (value: string): `${string}px` => `${value}px`; // TODO: remove appendPxSuffix - use appendUnit after fix
+
+// TODO: remove getValueInPx - use appendUnit after fix
```

Comment:
Fullfill the todo before merge

Reply:
This is not fixed yet, so it has to remain until it is fixed. I do not know who works on that.